ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -1,120 +1,110 @@
 presubmits:
   openshift/installer:
-  - name: pull-ci-origin-installer-unit
-    agent: kubernetes
-    context: ci/prow/unit
+  - agent: kubernetes
+    always_run: true
     branches:
     - master
-    rerun_command: "/test unit"
-    always_run: true
-    trigger: "((?m)^/test( all| unit),?(\\s+|$))"
+    context: ci/prow/unit
     decorate: true
+    name: pull-ci-origin-installer-unit
+    rerun_command: /test unit
     skip_cloning: true
     spec:
-      serviceAccountName: ci-operator
       containers:
-      - name: test
-        image: ci-operator:latest
-        resources: {"requests": {"cpu": "10m"},"limits":{"cpu":"500m"}}
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --target=unit
+        command:
+        - ci-operator
         env:
         - name: CONFIG_SPEC
           valueFrom:
             configMapKeyRef:
-              name: ci-operator-openshift-installer
               key: master.json
-        command:
-        - ci-operator
-        args:
-        - --artifact-dir=$(ARTIFACTS)
-        - --target=unit
-
-  - name: pull-ci-origin-installer-e2e-aws
-    agent: kubernetes
-    context: ci/prow/e2e-aws
+              name: ci-operator-openshift-installer
+        image: ci-operator:latest
+        name: test
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+      serviceAccountName: ci-operator
+    trigger: ((?m)^/test( all| unit),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
     branches:
     - master
-    rerun_command: "/test e2e-aws"
-    # The abomination below is equivilent to `^((?!Documentation).)*$`. Since
-    # Go doesn't support negative lookaheads, we are stuck with the following.
-    run_if_changed: "^([^D]|D(D|oD|ocD|ocuD|ocum(D|e(D|n(D|t(D|aD|atD|atiD|atioD)))))*([^Do]|o[^Dc]|oc[^Du]|ocu[^Dm]|ocum([^De]|e([^Dn]|n([^Dt]|t([^Da]|a[^Dt]|at[^Di]|ati[^Do]|atio[^Dn]))))))*(D(D|oD|ocD|ocuD|ocum(D|e(D|n(D|t(D|aD|atD|atiD|atioD)))))*(o|oc|ocu|ocum(e(n(t(a|at|ati|atio)?)?)?)?)?)?$"
```

Comment:
Striping the comment here makes the terrible regexp harder to understand ;).  Is it worth adding the comment back, or do we expect future mechanical YAML rewrites to continue to clobber comments?

Reply:
Oh, right, comments :/ We plan to put in place more tooling that interact with jobs but I think it's worth the effort to keep the comments there. I cannot simply put them back because we now have a check in place that would fail but I can fix the check as well. All changes will go through PRs and we do not plan any huge rewrites like this so we can keep the comments with just a little care.

Reply:
https://github.com/openshift/release/pull/1313